packages/amplify_authenticator/pubspec.yaml

```diff
@@ -1,16 +1,18 @@
 name: amplify_authenticator
-description: A new Flutter package project.
-version: 0.0.1
-homepage:
+description: A prebuilt sign in/sign up experience for Amplify Auth.
+version: 1.0.0-rc.0
```

Comment:
Setting this at `0.1.0` creates the following warnings:

```
Package validation found the following potential issues:
* Packages dependent on a pre-release of another package should themselves be published as a pre-release version. If this package needs amplify_auth_cognito version 0.3.0-rc.3, consider publishing the package as a pre-release instead.
  See https://dart.dev/tools/pub/publishing#publishing-prereleases For more information on pre-releases.
* Packages dependent on a pre-release of another package should themselves be published as a pre-release version. If this package needs amplify_auth_plugin_interface version 0.3.0-rc.3, consider publishing the package as a pre-release instead.
  See https://dart.dev/tools/pub/publishing#publishing-prereleases For more information on pre-releases.
* Packages dependent on a pre-release of another package should themselves be published as a pre-release version. If this package needs amplify_core version 0.3.0-rc.3, consider publishing the package as a pre-release instead.
  See https://dart.dev/tools/pub/publishing#publishing-prereleases For more information on pre-releases.
* Packages dependent on a pre-release of another package should themselves be published as a pre-release version. If this package needs amplify_flutter version 0.3.0-rc.3, consider publishing the package as a pre-release instead.
  See https://dart.dev/tools/pub/publishing#publishing-prereleases For more information on pre-releases.

Package has 4 warnings.
pub finished with exit code 65
```

Only other option would be `0.1.0-rc` but this is like double RC, and since the GA release is planned soon, `1.0.0-rc` seems appropriate.